sql/core/src/test/resources/sql-tests/inputs/subquery/in-subquery/in-null-semantics.sql

```diff
@@ -0,0 +1,57 @@
+create temp view v (c) as values (1), (null);
+create temp view v_empty (e) as select 1 where false;
+
+-- Note: tables and temp views hit different optimization/execution codepaths
+create table t(c int) using parquet;
```

Comment:
nit: if file format doesn't matter, probably json/csv is better as it's faster to create and read small tables.

Reply:
👍 will update